src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION = 0;
+    private static final int AMOUNT = 1;
```

Comment:
this is not operation per se, we should indicate this is index in order to avoid confusion
```suggestion
    private static final int OPERATION_INDEX = 0;
    private static final int AMOUNT_INDEX = 1;
```